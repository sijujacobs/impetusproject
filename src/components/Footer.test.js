import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../components/Footer';
describe('Footer', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Footer />);
  
    expect(component).toMatchSnapshot();
  });
});