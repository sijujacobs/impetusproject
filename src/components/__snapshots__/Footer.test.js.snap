// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly in "debug" mode 1`] = `
<div
  className="footer"
>
  <p>
    A Project on React  
    <span>
      Â© 
      2020
    </span>
  </p>
</div>
`;
